tests/ballerina-test/src/test/java/org/ballerinalang/test/services/nativeimpl/request/RequestNativeFunctionNegativeTest.java

```diff
@@ -201,8 +201,8 @@ public void testCompilationErrorTestCases() {
         Assert.assertEquals(resultNegative.getErrorCount(), 2);
         //testRequestSetStatusCode
         BAssertUtil.validateError(resultNegative, 0,
-                "undefined function 'setStatusCode' in struct 'ballerina/http:Request'", 4, 5);
+                "undefined function 'setStatusCode' in struct 'ballerina.http:Request'", 4, 5);
```

Comment:
Why has this been changed?

Reply:
Because that's the error message we get from the "PackageID" class's bvmAlias() method. We have no control over that message.